doc/content/developers.md

```diff
@@ -18,11 +18,12 @@ with NekRS. Therefore, the NekRS submodule points to the `cardinal` branch on ou
 of NekRS in your filesystem, first merge the latest master branch into a new NekRS feature update branch.
 
 ```
-$ pwd
-  /home/anovak/NekRS
-
-$ git co -b cardinal-update-5-21
-$ git merge origin/master
+pwd
+```
+An example output from the above command: `/home/anovak/NekRS`.
```

Comment:
I would prefer to leave it as the original case. This file isn't actually rendered in the documentation anyways.

Reply:
Set it back to the way it was.